### PR TITLE
Monitor Worker reports about form submissions should link to Submit History

### PR DESCRIPTION
**Pull request by nickpell**
fixes http://manage.dimagi.com/default.asp?171192#959451

@NoahCarnahan

### CODE REVIEW

File: corehq/apps/reports/standard/monitoring.py [hunk 1]
```diff
@@ -293,7 +326,7 @@ def get_number_cases(self, user_id, modified_after=None, modified_before=None, c
         )
         return qs.count()
 
-class SubmissionsByFormReport(WorkerMonitoringReportTableBase,
+class SubmissionsByFormReport(WorkerMonitoringFormReportTableBase,
```
Comment (dimagimon) [inline corehq/apps/reports/standard/monitoring.py]: E302 expected 2 blank lines, found 1

File: corehq/apps/reports/standard/monitoring.py [hunk 2]
```diff
@@ -596,7 +629,7 @@ def raw_user_link_url(self):
         from corehq.apps.reports.standard.inspect import SubmitHistory
         return SubmitHistory.get_url(domain=self.domain)
 
-class FormCompletionTimeReport(WorkerMonitoringReportTableBase, DatespanMixin,
+class FormCompletionTimeReport(WorkerMonitoringFormReportTableBase, DatespanMixin,
```
Comment (dimagimon) [inline corehq/apps/reports/standard/monitoring.py]: E302 expected 2 blank lines, found 1

File: corehq/apps/reports/standard/monitoring.py [hunk 3]
```diff
@@ -746,7 +753,7 @@ def get_data(users, group_by_user=True):
         return rows
 
 
-class FormCompletionVsSubmissionTrendsReport(WorkerMonitoringReportTableBase, MultiFormDrilldownMixin, DatespanMixin):
+class FormCompletionVsSubmissionTrendsReport(WorkerMonitoringFormReportTableBase, MultiFormDrilldownMixin, DatespanMixin):
```
Comment (dimagimon) [inline corehq/apps/reports/standard/monitoring.py]: E501 line too long (122 > 115 characters)

**Comment by NoahCarnahan** [comment]
:+1:

**Pull request merged**